.github/workflows/ci_cd_release.yml

```
@@ -232,9 +234,24 @@ jobs:
         skip-existing: false
         verbose: true
 
+    - name: "Generate link to migration guide for current release"
+      id: link-migration-guide
+      run: |
+        BASE_LINK="https://actions.docs.ansys.com/version/stable/migrations/index.html#version-v"
+        # Append version number to point to specific entry in the migration guide
+        MAJOR="${{ needs.rolling-release.outputs.tag_major }}"
+        LINK="${BASE_LINK}${MAJOR}"
+        echo "LINK=${LINK}" >> $GITHUB_OUTPUT
+
     - name: "Release to GitHub"
       uses: softprops/action-gh-release@da05d552573ad5aba039eaac05058a918a7bf631 # v2.2.2
       with:
+        body: >
+          If you are migrating the actions of your project to a new major release, breaking changes can be expected!
+
+          To make sure everything is compatible with your current workflow, please refer to the
+          [Migration Guide](${{ steps.link-migration-guide.outputs.LINK }}) to find out more about new features
+          and potential breaking changes!
```

```suggestion
          If you are migrating the actions of your project to a new major release, breaking changes can be expected.

          To make sure everything is compatible with your current workflow, refer to the
          [Migration Guide](${{ steps.link-migration-guide.outputs.LINK }}) to find out more about new features
          and potential breaking changes.
```

Just suggesting to remove the exclamation marks -- makes the text more neutral. Also, following @PipKat's recommendations we should avoid using "please" and similar wording when requesting users to do something.